travis-ci/vmtest/mkrootfs.sh

```diff
@@ -87,7 +87,34 @@ packages=(
 
 pacstrap -C "$pacman_conf" -cGM "$root" "${packages[@]}"
 
+makedeps=(
+	# iproute2
+	asp
+	bison
+	fakeroot
+        flex
+	gcc
+        gettext
+	make
+	patch
+	pkgconf
+)
+
+pacman -Sy --noconfirm "${makedeps[@]}"
+asp export iproute2
+chmod 777 iproute2
+
+cd iproute2
+sed -i 's/'"'"'iptables'"'"' //g' PKGBUILD
+sed -i '/makedepends/d' PKGBUILD
+chmod 666 *
+su nobody -s /bin/bash -c "makepkg -c --skippgpcheck"
```

Comment:
`makepkg` expects some env vars to be set, e.g. `BUILDDIR`, `PKGDEST`, `SRCDEST`.

Reply:
Did you hit issues with this?

Without these options, AFAICT `makepkg` just places artifacts into the current directory. With these options, it expects to create the specified directory and insert the artifacts there. I don't see the point in micromanaging all of these settings if the default is sane (and we'll clean up the files in the current directory).

Reply:
Hmm, I had no trouble with this on friday but now it keeps complaining. I'll dig a little deeper.

Reply:
Looks like it was the original $PWD permissions. I've changed it to move into $root first to create the build dir there, and fixed up related permissions. Should be good now.